### DISCUSSION TITLE
Add link to preview on edit page

### DISCUSSION THREAD

**Discussion by thickconfusion** [jonaswinkler/paperless-ng#1296] [Feature Requests]
On the dashboard, document link takes you to the edit page, which is usually my quick way to get a document.
![Dashboard - Pape](https://user-images.githubusercontent.com/46466294/132703131-bbfa5c26-a692-4943-9671-ff1e079b4dbf.png)

However, the edit page only shows a small preview when I want a full tab preview.
![Buyer_Builder_Plot_Plan2-Paperless-ng](https://user-images.githubusercontent.com/46466294/132702870-771e28cf-f01e-4a87-8c84-c4e27ebb3f5e.png)
The document page allows you to select the eyeball icon for a link to the preview.
![Documents-Paperless-ng](https://user-images.githubusercontent.com/46466294/132702922-ddc4674e-6118-4b23-afd1-b6adc62edd94.png)

**Comment by sukisoft** [comment]
Would come in very handy, yes. But I personally think, to be able to resize the preview dialog which opens when hovering the eyeball would be a good addition to your approach.

**Comment by iwishiwasaneagle** [comment]
Bump. I've been wanting to open the PDF quickly, but have to switch to the card view just to get the eyeball icon.
